assets/sass/v2/_footer.scss

```diff
@@ -0,0 +1,8 @@
+.siw-main-footer {
+  .auth-footer {
+    .js-cancel {
+      display: block;
+      text-align: right;
```

Comment:
Cannot just be align to right which doesn't work if there are other links to the left. e.g. at required password page.

<img width="431" alt="Screen Shot 2019-08-13 at 8 25 47 AM" src="https://user-images.githubusercontent.com/13139863/62954291-fb015c00-bda3-11e9-9b44-da6c57b564dc.png">

Reply:
yeah, it also has a display block on it, so the cancel link will actually be on its own line currently but might be better to float it right